packages/ckeditor5-dev-webpack-plugin/README.md

```diff
@@ -3,7 +3,7 @@ CKEditor 5 webpack plugin
 
 Intergrate [CKEditor 5](https://ckeditor.com)'s build process with your [webpack](https://webpack.js.org) setup.
 
-Requires webpack ^4.0.0.
+Requires webpack `>=4.0.0`.
```

Comment:
```suggestion
Requires webpack `^4.0.0` or `^5.0.0`.
```
Just to avoid confusion when webpack 6 will be ready and we miss the release day.